### PR TITLE
Use appropriate log levels, wrap errors coming from msgraph for logging

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

This PR takes into account [discussion](https://community-daily.mattermost.com/core/pl/6dxmx8ixrtyexn6hgfo8qps3pc) on logging conventions, to have the `Error` log level be specific to items the admin should take action on.

Additionally, any errors received directly from msgraph have been wrapped to point out that the specific error string came from msgraph, and the method name involved. This should make it easier for us to track down specific issues regarding msgraph requests.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-plugin-mscalendar/issues/88

### CODE REVIEW

**Comment by levb** [comment]
@mickmister not merging yet to give you a chance to address @larkox's comment

**Comment by mickmister** [comment]
@levb @larkox I meant to put the "Awaiting PR" label on this one on account of two big open PRs. I'll go through and resolve the inconsistencies all at once after they're merged.

**Comment by mickmister** [comment]
I've changed the log format to match the convention in mattermost-server, which is to do err=`error` within the message. I've also changed all calls to `errors.Errorf` to `fmt.Errorf` to match the convention in mattermost-server.

**Comment by mickmister** [comment]
@larkox Can you please take the log message format into account with your open PRs?

File: server/command/command.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func (c *Command) isValid() (subcommand string, parameters []string, err error)
 	split := strings.Fields(c.Args.Command)
 	command := split[0]
 	if command != "/"+config.CommandTrigger {
-		return "", nil, errors.Errorf("%q is not a supported command. Please contact your system administrator.", command)
+		return "", nil, fmt.Errorf("%q is not a supported command. Please contact your system administrator.", command)
```
Comment (levb) [inline server/command/command.go:103]: why `fmt`? just curious.

File: server/command/create_event.go [hunk 2]
```diff
@@ -82,13 +81,13 @@ func parseCreateArgs(args []string, timeZone string) (*remote.Event, error) {
...
 		if !flags[req] {
-			return nil, errors.Errorf("Missing required flag: `--%s` ", req)
+			return nil, fmt.Errorf("Missing required flag: `--%s` ", req)
 		}
 	}
 
 	help, err := createFlagSet.GetBool("help")
 	if help == true {
-		return nil, errors.Errorf(getCreateEventFlagSet().FlagUsages())
+		return nil, fmt.Errorf(getCreateEventFlagSet().FlagUsages())
```
Comment (larkox) [inline server/command/create_event.go:90]: Any strong reason to use the fmt.Errorf instead of just errors.New?

**Pull request merged**